atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TodoEteTest.java

```diff
@@ -13,39 +13,35 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.palantir.atlasdb.ete.todo;
+package com.palantir.atlasdb.ete;
 
+import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.core.IsNot.not;
+import static org.hamcrest.core.IsNull.nullValue;
 
 import org.junit.Test;
 
-import com.palantir.atlasdb.ete.EteSetup;
 import com.palantir.atlasdb.todo.ImmutableTodo;
 import com.palantir.atlasdb.todo.Todo;
 import com.palantir.atlasdb.todo.TodoResource;
 import com.palantir.timestamp.TimestampService;
 
-public abstract class TodoEteTest extends EteSetup {
+public class TodoEteTest {
     private static final Todo TODO = ImmutableTodo.of("some stuff to do");
 
-    protected abstract TimestampService createTimestampClient();
-
     @Test
     public void shouldBeAbleToWriteAndListTodos() {
-        TodoResource todoClient = createClientToSingleNode(TodoResource.class);
+        TodoResource todoClient = EteSetup.createClientToSingleNode(TodoResource.class);
 
         todoClient.addTodo(TODO);
         assertThat(todoClient.getTodoList(), contains(TODO));
     }
 
     @Test
-    public void
-    shouldExposeATimestampServer() {
-        TimestampService timestampClient = createTimestampClient();
+    public void shouldExposeATimestampServer() {
+        TimestampService timestampClient = EteSetup.createClientToAllNodes(TimestampService.class);
```

Comment:
Minor behavior change for `CassandraNoLeaderTodoEteTest` as the client is now wrapped with `failoverFeignTarget`. I don't see this affecting the testing behavior, but should confirm.